core/che-core-api-dto/src/test/java/org/eclipse/che/dto/definitions/DTOHierarchy.java

```diff
@@ -60,4 +62,6 @@
         void setShadowedField(GrandchildDto v);
     }
 
+    public interface GrandchildWithoutDto extends Child, Closeable {
```

Comment:
Also Mb you are able to reuse just `Child` class or create `Grandchild` class if you want three level of extending